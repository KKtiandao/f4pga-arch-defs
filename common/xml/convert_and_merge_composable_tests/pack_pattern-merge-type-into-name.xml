<?xml version="1.0"?>
<xml>
 <pb_type>
  <direct>
   <pack_pattern name="B" type="carry" />
  </direct>
  <mux>
   <pack_pattern name="A" type="pack">
    <innertag />
   </pack_pattern>
  </mux>
 </pb_type>
</xml>
