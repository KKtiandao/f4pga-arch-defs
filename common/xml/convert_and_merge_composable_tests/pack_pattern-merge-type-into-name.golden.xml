<?xml version="1.0"?>
<xml>
  <pb_type>
    <direct>
      <pack_pattern name="carry-B"/>
    </direct>
    <mux>
      <pack_pattern name="pack-A">
        <innertag/>
      </pack_pattern>
    </mux>
  </pb_type>
</xml>
